src/keystore.h

```diff
@@ -44,11 +45,21 @@ class CKeyStore
     virtual bool RemoveWatchOnly(const CScript &dest) =0;
     virtual bool HaveWatchOnly(const CScript &dest) const =0;
     virtual bool HaveWatchOnly() const =0;
+
+    //! Add a spending key to the store.
+    virtual bool AddSpendingKeyPaymentAddress(const libzcash::SpendingKey &key, const libzcash::PaymentAddress &address) =0;
```

Comment:
I added this method to parallel the existing methods for transparent addresses. I will look further into why that split was made for them.